Samples/Cordova/ExampleProject/platforms/android/android.json

```diff
@@ -63,9 +63,9 @@
   },
   "installed_plugins": {
     "clevertap-cordova": {
-      "CLEVERTAP_ACCOUNT_ID": "67Z-RRK-696Z",
-      "CLEVERTAP_TOKEN": "322-1a6",
-      "CLEVERTAP_REGION": "sk1-staging-25",
+      "CLEVERTAP_ACCOUNT_ID": "TEST-4RW-944-876Z",
```

Comment:
We should not push any credentials to master

Reply:
@Anush-Shand could you reverse this?

Reply:
I will approve this PR, just ensure we don't push any CT credentials to master in the final PR